Configurateur/front/lib/components/custom_header.dart

```diff
@@ -0,0 +1,260 @@
+                      onChanged: (bool value) {
+                        Provider.of<ThemeService>(context, listen: false)
+                            .switchTheme();
+                        setState(() {});
+                      },
+                      activeColor: darkTheme.primaryColor,
+                      inactiveTrackColor: lightTheme.primaryColor,
+                    )
+                  ],
+                ),
+              ),
+              PopupMenuButton<String>(
+                tooltip: "Authentification",
+                icon: Icon(
+                  size: 35,
+                  Icons.account_circle,
+                  color: Provider.of<ThemeService>(context).isDark
+                      ? darkTheme.primaryColor
+                      : lightTheme.primaryColor,
+                ),
+                itemBuilder: (BuildContext context) {
+                  List<PopupMenuEntry<String>> items = [];
+                  if (token == '') {
+                    items.addAll([
+                      const PopupMenuItem<String>(
+                        value: 'connexion',
+                        child: Text(
+                          'Connexion',
+                          style: TextStyle(
+                            color: Color(0xff033F63),
+                          ),
+                        ),
+                      ),
+                      const PopupMenuItem<String>(
+                        value: 'inscription',
+                        child: Text(
+                          'Inscription',
+                          style: TextStyle(
+                            color: Color(0xff033F63),
+                          ),
+                        ),
+                      ),
+                    ]);
+                  } else {
+                    items.add(
+                      const PopupMenuItem<String>(
+                        value: 'profil',
+                        child: Text('Profil',
+                            style: TextStyle(
+                              color: Color(0xff033F63),
+                            )),
+                      ),
+                    );
+                    if (userMail == "risu.admin@gmail.com") {
+                      items.add(
+                        const PopupMenuItem<String>(
+                          value: 'admin',
+                          child: Text(
+                            'Administration',
+                            style: TextStyle(
+                              color: Color(0xff033F63),
+                            ),
+                          ),
+                        ),
+                      );
+                    }
+                    items.add(
+                      const PopupMenuItem<String>(
+                        value: 'disconnect',
+                        child: Text('Déconnexion',
+                            style: TextStyle(
+                              color: Colors.red,
+                            )),
```

Comment:
Maybe add , after the first ) so that the file is more readable

Reply:
done